pyinstaller-gui.py

```diff
@@ -76,7 +85,20 @@ def killapp(self, event):
         sys.exit(0)
 
     def makePackage(self, event):
-        commands = ['python3', 'pyinstaller.py']
+        if pyversion == 2:
+            commands = ['python', 'pyinstaller.py']
+
+        elif pyversion[0] == 3:
+            if sys.platform.startswith('win'):
+                if pyversion[1] > 2:  # python3.3+
+                    commands = ['py', '-3', 'pyinstaller.py']
+                else:
+                    commands = [sys.executable, 'pyinstaller.py']
+            else:
+                commands = ['python3', 'pyinstaller.py']
+        else:
```

Comment:
Couldn't this be simplified by using sys.executable in all cases except `sys.platform.startswith('win') and pyversion >= (3,3)`?